e2e/wdio/platform/tests/time-picker.e2e-spec.ts

```diff
@@ -0,0 +1,114 @@
+import {
+    click,
+    doesItExist,
+    elementArray,
+    getAttributeByName,
+    getElementArrayLength,
+    getText,
+    getValue,
+    refreshPage,
+    scrollIntoView,
+    sendKeys,
+    setValue,
+    waitForElDisplayed,
+    waitForPresent
+} from '../../driver/wdio';
+import { time, text, defaultValidTime } from '../fixtures/testData/time-picker';
+import { TimePicker } from '../pages/time-picker.po';
+
+let timePickerPage: TimePicker;
+const {
+    activeTimePickerInput, timePickerButton, timePickerInput, timerExpanded,
+    activeTimePickerButton, errorBorder,
+    selectedHours, selectedMinutes, period,
+    navigationDownArrowButton, timeItem, setToNullButton, setValidTimeButton
+} = new TimePicker();
```

Comment:
why PO import done like this instead of:
const timePickerPage = new TimePicker();
const { ... } = timePickerPage; 
(in the 1st describe)